packages/utilities/components/compose.ts

```diff
@@ -0,0 +1,43 @@
+export function compose<A, B, C>(b: (v: B) => C, a: (v: A) => B): (a: A) => C;
+export function compose<A, B, C, D>(
+  c: (v: C) => D,
+  b: (v: B) => C,
+  a: (v: A) => B
+): (a: A) => D;
+export function compose<A, B, C, D, E>(
+  d: (v: D) => E,
+  c: (v: C) => D,
+  b: (v: B) => C,
+  a: (v: A) => B
+): (a: A) => E;
+export function compose<A, B, C, D, E, F>(
+  e: (v: E) => F,
+  d: (v: D) => E,
+  c: (v: C) => D,
+  b: (v: B) => C,
+  a: (v: A) => B
+): (a: A) => F;
+export function compose<A, B, C, D, E, F, G>(
+  f: (v: F) => G,
+  e: (v: E) => F,
+  d: (v: D) => E,
+  c: (v: C) => D,
+  b: (v: B) => C,
+  a: (v: A) => B
+): (a: A) => G;
+export function compose<A, B, C, D, E, F, G, H>(
+  g: (v: G) => H,
+  f: (v: F) => G,
+  e: (v: E) => F,
+  d: (v: D) => E,
+  c: (v: C) => D,
+  b: (v: B) => C,
+  a: (v: A) => B
+): (a: A) => H;
+
+export function compose(...functions) {
```

Comment:
Why do you have a named and a default export of the very same thing?

Reply:
So that I can use both notations for import 
`import compose from "compose.ts"` and
`import {compose} from "compose.ts"`

Reply:
Here is a quick example: https://codesandbox.io/s/231wklxny0?expanddevtools=1